tests/unit_tests/test_game.py

```diff
@@ -91,6 +92,13 @@ def test_add_game_connection_throws_if_not_lobby_state(game: Game, players, mock
 
     game.on_game_end.assert_any_call()
 
+@patch('asyncio.sleep', return_value=None)
+async def test_game_marked_dirty_when_timed_out(game: Game):
+    game.state = GameState.INITIALIZING
+    await game.timeout_game()
+    assert game.state == GameState.ENDED
+    assert game in game.game_service.dirty_games
+
```

Comment:
In the future, please seperate `arrange`, `act` and `assert` parts of the test for readability. We try and follow this pattern elsewhere in the code. Otherwise -- nice job on this 👍